docs/reference/data-streams/lifecycle/tutorial-manage-data-stream-retention.asciidoc

```diff
@@ -0,0 +1,184 @@
+
+[source,console-result]
+--------------------------------------------------
+{
+  "data_streams": [
+    {
+      "name": "my-data-stream",                                   <1>
+      "lifecycle": {
+        "enabled": true                                           <2>
+      }
+    }
+  ]
+}
+--------------------------------------------------
+// TESTRESPONSE[skip:the result is for illustrating purposes only]
+<1> The name of your data stream.
+<2> Ensure that the lifecycle is enabled, meaning this should be `true`.
+
+. <<what-is-retention>>
+. <<retention-configuration>>
+. <<effective-retention-calculation>>
+. <<effective-retention-application>>
+
+[discrete]
+[[what-is-retention]]
+==== What is data stream retention?
+
+We define retention as the least amount of time the data of a data stream are going to be kept in {es}. After this time period
+has passed, {es} is allowed to remove this data to free up space and/or manage costs.
+
+NOTE: Retention does not define the period that the data will be removed, but the minimum time period they will be kept.
+
+We define 4 different types of retention:
+
+* The data stream retention, or `data_retention`, which is the retention configured on the data stream level. It can be
+set via an <<index-templates,index template>> for future data streams or via the <<data-streams-put-lifecycle, set data
+stream lifecycle API>> for an existing data stream. When the data stream retention is not set, it implies that the data
+needs to be kept forever.
+* The global default retention, or `default_retention`, which is a retention configured on a project level and will be
+applied to all data streams managed by data stream lifecycle that do not have `data_retention` configured. Effectively,
+it ensures that there will be no data streams keeping their data forever. This can be set via the
+<<data-streams-put-global-retention, set global retention API>>.
+* The global max retention, or `max_retention`, which is a retention configured on a project level and will be applied to
+all data streams managed by data stream lifecycle. Effectively, it ensures that there will be no data streams whose retention
+will exceed this time period. This can be set via the <<data-streams-put-global-retention, PUT global retention API>>.
+* The effective retention, or `effective_retention`, which is the retention applied at a data stream on a given moment.
+Effective retention cannot be set, it is derived by taking into account all the configured retention listed above and is
+calculated as it is described <<effective-retention-calculation,here>>.
+
+[discrete]
+[[retention-configuration]]
+==== How to configure retention?
+
+`data_retention`
+The data stream level retention can be configured in two ways:
+- For new data streams, it can be defined in the index template that would be applied during the data stream's creation.
+You can use the <<indices-put-template,create index template API>>, for example:
++
```

Comment:
Is this line supposed to be here? It just renders a `+` on a line of its own, which is probably not your intention.

Reply:
No indeed, when the list is fixed it works as intended. When you have list it indents its paragraphs the `+` allows you to continue in this indentation even when you have a blank line.